crates/extensions/tedge_config_manager/src/config.rs

```diff
@@ -24,6 +25,7 @@ use tedge_utils::file::PermissionEntry;
 
 use super::error::InvalidConfigTypeError;
 
+pub const DEFAULT_PLUGIN_DIR: &str = "config-plugins";
```

Comment:
Why do we need this constant while there is already a default for `configuration.plugin_paths`?

Reply:
Resolved by a16fff1cbf